stable/mariadb/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mariadb
-version: 6.2.1
+version: 6.2.2
```

Increase minor version:
```suggestion
version: 6.3.0
```